drupal-8/.gitignore

```diff
@@ -0,0 +1,29 @@
+# Ignore directories generated by Composer
+/drush/contrib/
+/vendor/
+/web/core/
+/web/modules/contrib/
+/web/themes/contrib/
+/web/profiles/contrib/
+/web/libraries/
```

Comment:
I don't think we installed drupal in the right place. I would expect a .gitignore file to be in the site root, and then this file would be telling github to not pay attention to all of these drupal files (so the actual code review would be much smaller).

Reply:
I can move the .`gitignore` file to the root. I created everything inside the `drupal-8` folder as we also have a `weekly ship` folder and did not want to conflict with that.

Reply:
Just moving the .gitignore won't do it, as the other paths will still be off-by-one-directory.  I think we need 
```
/web/modules/contrib/
/web/themes/contrib/
/web/profiles/contrib/
/web/libraries/
```
etc to be siblings of weekly-ship
They shouldn't overwrite it, they should just drop in alongside. And if they DO overwrite it, then we'll trash the branch and figure something else out :-)

Reply:
I think we need to add to the `.gitignore` this didn't really lessen the files here at all. 
These last level dir don't exist `/contrib`
```
/web/modules/contrib/
/web/themes/contrib/
/web/profiles/contrib/
/web/libraries/
```

Reply:
👍 Depending on what we do next, we MAY also be able to just say /web/* and capture _whatever_. But then we may find ourselves wanting to put something in web and having to unwind that. So your call :-)

Reply:
Scratch that. web has a bunch of stuff we'll want to check in, I think.